README.md

```diff
@@ -11,6 +11,10 @@ This repository includes files with plain SQL that can be used to recreate a dat
 - Use [data.sql](./data.sql) to populate tables with sample data.
 - Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
 
+## Database diagram
+
+ <img src="./vet_clinic_diagram.png" alt="database diagram" width="200"  height="auto" />
+
 ## Authors
 
```

Comment:
- [ ] [optional] Kindly ensure to use the correct readme template to create your readme file, here is the recommended microverse [readme template](https://github.com/microverseinc/readme-template)